guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -17,6 +16,31 @@ The release notes include:
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## v2002.0.22
+
+-  {:.new}**Changes to the ece-tools composer package**
+
+   We changed the structure of the {{ site.data.var.ct }} package to reduce complexity for scheduling updates and working with community contributions.
+
+   -  {:.new}{{ site.data.var.ece }} patches are no longer included in the {{ site.data.var.ct }} ece-tools package to allow Magento to manage and deliver patches independently without requiring an {{ site.data.var.ct }} package release. These patches are now delivered in a separate Magento Cloud Patches module that is a required dependency for the {{ site.data.var.ct }} package.
```

Comment:
```suggestion
   -  {:.new}{{ site.data.var.ece }} patches are no longer included in the {{ site.data.var.ct }} package to allow Magento to manage and deliver patches independently without requiring an {{ site.data.var.ct }} package release. These patches are now delivered in a separate Magento Cloud Patches package that is a required dependency for the {{ site.data.var.ct }} package.
```

You are calling the MCP a module. In the repos and other places it is a package that is part of the Magento Cloud Suite.

I wonder if the use of independent and dependent in the same block may cause confusion. Could probably remove `independently`.

Reply:
I agree on `module` vs `package` naming - let's use `package`.
I am not sure about dependency wording though - logically new package is used to release patches independently but ECE-Tools is dependent on it. So even though it may sound confusing, it's the correct content as for me.

Reply:
so part of the confusion is that we use module for https://github.com/magento/magento-cloud-components -- is there a reason to differentiate that component as a module vs. the others as packages?

Reply:
Yes. Components by its nature is a Magento 2 module, directly extending core. All other 3 (ECE-Tools / Patches / Docker) are better described as packages, since they have no direct core dependency.